src/Persistence/IsProxy.php

```diff
@@ -65,7 +66,10 @@ public function _refresh(): static
         $this->initializeLazyObject();
         $object = $this->lazyObjectState->realInstance;
 
-        Configuration::instance()->persistence()->refresh($object);
+        try {
+            Configuration::instance()->persistence()->refresh($object);
+        } catch (PersistenceNotAvailable) {
+        }
```

Comment:
not sure you'll be OK for this, but, WDYT? `_refresh()` is called on every call on the original class. This would prevent any error when proxies are used in a unit test.

I've not added the same try/catch in other proxy methods like `_save()` or `_delete()` since it would not be normal to call these methods in unit test

Reply:
Should we perhaps move this logic to another private method that is called by every original class method?

I feel like if someone is calling `->_refresh()`, we should throw the exception.

Reply:
sounds good